base/task/sequence_manager/thread_controller_with_message_pump_impl.cc

```diff
@@ -335,6 +335,9 @@ ThreadControllerWithMessagePumpImpl::DoWork() {
           : WorkDeduplicator::NextTask::kIsDelayed;
   if (work_deduplicator_.DidCheckForMoreWork(next_task) ==
       ShouldScheduleWork::kScheduleImmediate) {
+    recordreplay::Assert("[RUN-548] ThreadControllerWithMessagePumpImpl::DoWork #5 %ld",
```

Comment:
Why is this labeled with RUN-548 instead of RUN-1124?  Looked around and while 548 has some frames with ThreadControllerWithMessagePumpImpl on stack, it seems less relevant.

Reply:
It's hard to say, we suddenly started seeing some crashes with an assertion that was originally associated with RUN-548.  I suspect the two issues are the same underlying problem however, both of these signatures started appearing recently at the same time for some reason.

Reply:
Er, actually this change isn't supposed to be here at all, it's from another PR I was working on that git picked up.  I'll remove this change.